src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2315,7 +2315,7 @@ private BoundExpression BindExplicitNullableCastFromNonNullable(ExpressionSyntax
                 var underlyingExpr = BindCastCore(node, operand, underlyingTargetTypeWithAnnotations, wasCompilerGenerated: false, diagnostics: bag);
                 if (underlyingExpr.HasErrors || bag.HasAnyErrors())
                 {
-                    Error(diagnostics, ErrorCode.ERR_NoExplicitConv, node, operand.Type, targetTypeWithAnnotations.Type);
+                    Error(diagnostics, getErrorCode(bag), node, operand.Type, targetTypeWithAnnotations.Type);
```

Comment:
Consider inlining

Reply:
You say to do this?
```csharp
 Error(diagnostics,  (ErrorCode)diagnostic.AsEnumerable().Last().Code, node, operand.Type, targetTypeWithAnnotations.Type);
```

Reply:
Sorry I should have been clearer. I think you should create a `var errorCode` and pass it to the Error method instead of using a local function.